lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -46,6 +50,7 @@
                        _.each([
                            [':default', _.bind(this.routeNotFound, this)],
                            ['topics/:topic_id', _.bind(this.browseTopic, this)],
+                           ['topics/:topic_id/create_new_team(/)', _.bind(this.newTeam, this)],
```

Comment:
Also, what is the purpose of the trailing `'(/)'`? We don't have it for any of our other URLs.